Week-1/Asignment.md

```diff
@@ -0,0 +1,130 @@
+# Week 1 Assignment 1
+
+
+So, When I enter the url in the browser what happens,also how browser fetch the desired result
+
+ A.  So, Basically when I type the url into any browser what I want is that i want to reach the server where the website is hosted.
+
+So, now what the browser does is that it will look for the IP address of the domain name in DNS ,Domain Name server.
+
+so here we can also access the website by typing the ip address but it is difficult to remember the nubers of the IP address so what we do is we will remember the name of the website with IP addressby the DNS.
+
+**Now what does DNS do?**
+
+DNS will checks  the cache part to run DNS query. Now It check with four cache type:
+BRowser Cache,OS cache ,Router CAche ,ISP cache.
+So, when start with browser cache if it does not contain cache then it will go for OS cache similarly in this manner if one not works it will go for another cache check.And if at the end ISP not work then it will go for DNS Recursive search and then initiate DNS query and commnicate with DNS server to find the IP address.
```

Comment:
Once browser has right query details from DNS. Where is that query data stored ?